addon/components/ember-th/component.js

```diff
@@ -39,95 +32,84 @@ const COLUMN_REORDERING = 2;
   @yield {object} columnValue - The column definition
   @yield {object} columnMeta - The meta object associated with this column
 */
-@tagName('th')
-export default class EmberTh extends BaseTableCell {
-  layout = layout;
+
+export default BaseTableCell.extend({
+  layout,
+  tagName: 'th',
+  classNameBindings: ['isSortable', 'isResizable', 'isReorderable'],
 
   /**
     The API object passed in by the table row
+    @argument
+    @required
+    @type('object')
   */
-  @argument
-  @required
-  @type('object')
-  api;
+  api: null,
 
   /**
     Action sent when the user clicks right this element
+    @argument
+    @type(optional(Action))
   */
-  @argument
-  @type(optional(Action))
-  onContextMenu;
+  onContextMenu: null,
 
-  @readOnly('api.columnValue')
-  columnValue;
+  columnValue: readOnly('api.columnValue'),
 
-  @readOnly('api.columnMeta')
-  columnMeta;
+  columnMeta: readOnly('api.columnMeta'),
 
   /**
     Any sorts applied to the table.
   */
-  @readOnly('api.sorts')
-  sorts;
+  sorts: readOnly('api.sorts'),
 
   /**
     Whether or not the column is sortable. Is true IFF the column is a leaf node
     onUpdateSorts is set on the thead.
   */
-  @className
-  @readOnly('columnMeta.isSortable')
-  isSortable;
+  isSortable: readOnly('columnMeta.isSortable'),
 
   /**
     Indicates if this column can be resized.
   */
-  @className
-  @readOnly('columnMeta.isResizable')
-  isResizable;
+  isResizable: readOnly('columnMeta.isResizable'),
 
   /**
    Indicates if this column can be reordered.
   */
-  @className
-  @readOnly('columnMeta.isReorderable')
-  isReorderable;
+  isReorderable: readOnly('columnMeta.isReorderable'),
 
-  @attribute('colspan')
-  @readOnly('columnMeta.columnSpan')
-  columnSpan;
+  columnSpan: readOnly('columnMeta.columnSpan'),
```

Comment:
This needs to be added to `attributeBindings`

Reply:
Good catch!